bazel/repository_locations.bzl

```diff
@@ -62,6 +67,11 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "nanopb-f8ac463766281625ad710900479130c7fcb4d63b",
         urls = ["https://github.com/nanopb/nanopb/archive/f8ac463766281625ad710900479130c7fcb4d63b.tar.gz"],
     ),
+    com_github_nghttp2_nghttp2 = dict(
+        sha256 = "42fff7f290100c02234ac3b0095852e4392e6bfd95ebed900ca8bd630850d88c",
+        strip_prefix = "nghttp2-1.33.0",
```

Comment:
@htuch nghttp2 version 1.35.0 is out if you would like to bump to that version. Master is currently set at 1.34.0
`SHA256` for 1.35.0 is `ea04e4e749df13d60a88b706752e04f6f907227bf380aefd7aeeb4aa0db43407`

Reply:
Yep, thanks, already bumped the versions.